mainframer.sh

```diff
@@ -89,7 +96,7 @@ function syncBeforeRemoteCommand {
 	eval "$COMMAND"
 
 	endTime="$(date +%s)"
-	echo "Sync done: took $((endTime-startTime)) seconds."
+	echo "Sync done: took $(convertsecs $((endTime-startTime)))"
```

Comment:
Of course it should :/! Will definitely provide a fix for this to make it work today :o